### ISSUE TITLE
Should we rename instance event in camalCase?

### ISSUE THREAD

**Issue by stop2stare** [SSENSE/vue-carousel#169]
We currently have event `pageChange` on component `Carousel`. But we also have event `paginationclick` on `Pagination` and `navigationclick` on `Navigation`. Should we rename those two event in camalCase like `pageChange`?

**Comment by quinnlangille** [comment]
Hmm yeah, I agree that this is a good idea. However, like we did with the implementation of `pagechange` -> `pageChange`, it would need to be backward compatible so we don't break previous installs.

**Comment by stop2stare** [comment]
Of course, we don't want breaking changes. But these two kinds of event names really make me confused.🤣
BTW, we're about releasing `v1.0.0`. Will we be able to change this by then?

**Comment by quinnlangille** [comment]
I think it's safer if we include a deprecation notice with `v1.0.0` and schedule actual removal for a later version, just to give devs some time to adjust

**Comment by stop2stare** [comment]
Sure thing.
